backend/src/main/java/de/neuefische/cgnjava222/productgallery/model/Product.java

```diff
@@ -5,7 +5,7 @@
 public record Product(String id,
```

Comment:
String id zur optik in einer neue Zeile und dann die Attribute. Gefolgt von einer neuen Zeile und dann schließen